keras_retinanet/models/mobilenet.py

```diff
@@ -83,13 +83,8 @@ def mobilenet_retinanet(num_classes, backbone='mobilenet224_1.0', inputs=None, *
 
     mobilenet = MobileNet(input_tensor=inputs, alpha=alpha, include_top=False, pooling=None, weights=None)
 
-    # get last layer from each depthwise convolution blocks 3, 5, 11 and 13
-    outputs = [mobilenet.get_layer(name='conv_pw_{}_relu'.format(i)).output for i in [3, 5, 11, 13]]
-
-    # create the mobilenet backbone
-    mobilenet = keras.models.Model(inputs=inputs, outputs=outputs, name=mobilenet.name)
-
     # create the full model
-    model = retinanet.retinanet_bbox(inputs=inputs, num_classes=num_classes, backbone=mobilenet, **kwargs)
+    model = retinanet.retinanet_bbox(inputs=inputs, num_classes=num_classes, backbone=mobilenet,
```

Comment:
I don't think passing the layers is a bad idea, but maybe it would be nicer if you pass the layer outputs themselves and not the names of the layers?

Reply:
That would be possible. In that case, it maybe even makes sense to only pass the output layers. Will the `backbone` value be used for anything by itself?

Reply:
Personally the names of the layers seems fine with me, they're a bit easier to manage that passing actual layer objects around. Support future improvements such as config loading etc. They're user defined alongside the backbone, so it should be fine. Passing layer objects around would require a bit of boilerplate prior to the `retinanet_bbox` call to manipulate the backbone object into giving the selected layers. 9 times out of 10, I imagine people will iterate until `name = X` and yield.
What advantages would passing the layer/layer outputs over names provide?

Reply:
We can add a utility function to extract layers by name if that is a problem. The simplest API is to take the objects we're interested in, not their identifiers in some other object.